packages/framework-integration-tests/package.json

```diff
@@ -78,7 +78,7 @@
     "eslint-plugin-unicorn": "~44.0.2"
   },
   "engines": {
-    "node": ">=18.0.0 <19.0.0"
+    "node": ">=18.12.0 <19.0.0"
```

Comment:
Is this a requirement of the new CDK version? Otherwise, I'd leave it as 18.0.0 in order to be less restrictive for end users.

Reply:
@javiertoledo [yes, it seems to be a requirement for the new cdk tf version, see here](https://developer.hashicorp.com/terraform/cdktf/release/upgrade-guide-v0-19#:~:text=we%20updated%20our%20minimum%20compatible%20Node.js%20version%20to%2018.12.). If it presents an issue, we can go back to version 0.17 I believe

Reply:
Gotcha, no objections then!